openedx/core/djangoapps/programs/utils.py

```diff
@@ -105,3 +105,35 @@ def get_programs_for_dashboard(user, course_keys):
             log.exception('Unable to parse Programs API response: %r', program)
 
     return course_programs
+
+
+def get_programs_for_credentials(user, programs_credentials, credential_configuration):
```

Comment:
Why does the credential configuration need to be passed as an argument to this function?